tests/create-projects/create-projects_test.go

```diff
@@ -0,0 +1,77 @@
+package create_projects
+
+import (
+    "fmt"
+    projectClientV1 "github.com/openshift/client-go/project/clientset/versioned/typed/project/v1"
+    "gotest.tools/assert"
+    meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/client-go/tools/clientcmd"
+    "os"
+    "os/exec"
+    "path"
+    "path/filepath"
+    "runtime"
+    "testing"
+)
+
+func homeDir() string {
+    if h := os.Getenv("HOME"); h != "" {
+        return h
+    }
+    return os.Getenv("USERPROFILE") // windows
+}
+
+func TestCreateProjectWithoutProjectId(t *testing.T) {
+    _, filename, _, _ := runtime.Caller(0)
+    dir := path.Join(path.Dir(filename), "..", "..")
+    cmd := exec.Command("sh", fmt.Sprintf("%s/create-projects/create-projects.sh", dir))
+    out, err := cmd.CombinedOutput()
+    if err == nil {
+        t.Errorf("Execution of `create-project.sh` must fail if no PROJECT_ID is set: %s", out)
+    }
+}
+
+func TestCreateProject(t *testing.T) {
+    _, filename, _, _ := runtime.Caller(0)
+    dir := path.Join(path.Dir(filename), "..", "..")
+    cmd := exec.Command("sh", fmt.Sprintf("%s/create-projects/create-projects.sh", dir))
+    cmd.Env = append(os.Environ(), "PROJECT_ID=unitt")
+    out, err := cmd.CombinedOutput()
+    if err != nil {
+        t.Errorf("Execution of `create-project.sh` failed: %s\n%s", err.Error(), out)
+    }
+    config, err := clientcmd.BuildConfigFromFlags("", filepath.Join(homeDir(), ".kube", "config"))
+    if err != nil {
+        t.Errorf("Cannot load cluster configuration: %s", err.Error())
```

Comment:
FYI `t.Errorf` spits out an error, but it does not stop the test. I think you probably want to use `t.Fatalf` instead.